codice/alliance
# CAL-440: Admin Console and Intrigue CSRF Protections

#### What does this PR do?
Adds the X-Requested-With header to outgoing requests for UIs that access jolokia to comply with the new CSRF protections soon to be added to DDF.

#### Who is reviewing it? 
@djblue
@rzwiefel
@tbatie

#### Choose 2 committers to review/merge the PR.
@andrewkfiedler
@shaundmorris 

#### How should this be tested?
Use Alliance-specific UIs and watch network requests to make sure no 403s are returned.

#### What are the relevant tickets?
https://github.com/codice/ddf/pull/3297
https://codice.atlassian.net/browse/DDF-3855
[CAL-440](https://codice.atlassian.net/browse/CAL-440)

#### Checklist:
